lib/SPIRV/SPIRVInternal.h

```diff
@@ -919,9 +922,11 @@ std::string getSPIRVFriendlyIRFunctionName(OCLExtOpKind ExtOpId,
 /// \param OC opcode of corresponding built-in instruction. Used to gather info
 /// for unsigned/constant arguments.
 /// \param Types of arguments of SPIR-V built-in function
+/// \param Operands of SPIRVInstruction
```

Comment:
```suggestion
/// \param Ops Operands of SPIRVInstruction
```
(I'm aware the previous line isn't using `\param` correctly either, but that's outside of this PR's scope.)

Reply:
done